docroot/modules/custom/prisoner_hub_sub_terms/src/Resource/SubTerms.php

```diff
@@ -91,7 +97,7 @@ public function process(Request $request, TermInterface $taxonomy_term): Resourc
     $query->groupBy('field_moj_series');
 
     // Aggregate the groupings by the most recently created, and sort by that.
-    $query->sortAggregate('changed', 'MAX', 'DESC');
+    $query->sortAggregate('created', 'MAX', 'DESC');
```

Comment:
As we are only invalidating on content inserts, I think it makes sense to switch the sorting order to be by created date.